example/vetted.py

```diff
@@ -0,0 +1,15 @@
+from fickling.pickle import Pickled
+import pickle
+import numpy as np
+
+
+arr = np.array([1,2,3])
+fickled = Pickled.load(pickle.dumps(arr))
+fickled.vetted_dependencies = ["numpy"]
```

Comment:
`numpy` itself is not safe. For example: https://numpy.org/doc/stable/reference/generated/numpy.distutils.exec_command.html

How are `vetted_dependencies` and `vetted_calls` intended to interact? Are `ndarray` and `dtype` the _only_ calls within `numpy` that are allowed? This should be documented somewhere in the README. I'm happy to help with that.